pkcs11/signer.go

```diff
@@ -169,7 +169,8 @@ func (s *signer) SignX509Cert(ctx context.Context, cert *x509.Certificate, keyId
 	defer pool.put(signer)
 	// Validate the cert request to ensure it matches the keyType and also the HSM supports the signature algo.
 	if val := isValidCertRequest(cert, signer.signAlgorithm()); !val {
-		log.Printf("SignX509Cert: unsupported signature algo %d, supported algo %d", cert.SignatureAlgorithm, signer.signAlgorithm())
+		log.Printf("signX509cert: CN '%s' unsupported signature algo %d, supported signature algo %d",
```

Comment:
Could we use `%q` and `SignatureAlgorithm.String()` as well as make the format Splunk-friendly?